packages/optimizer/README.md

```diff
@@ -216,14 +273,13 @@ get an intrinsic layout. For image detection to work, an optional dependency
 
 #### `minify`
 
-Minifies the generated HTML output and inlined CSS.
+Minifies the generated HTML output. Requires the following `terser` to be installed when inlined `amp-script` should be minified. Combine with [separateKeyframes](#separateKeyframes) to also minify CSS.
 
 - name: `minify`
 - valid options: `[true|false]`
-- default: `true`
+- default: `false`
 - used by: [MinifyHtml](lib/transformers/MinifyHtml.js), [SeparateKeyframes](lib/transformers/SeparateKeyframes.js)
-
-**Warning:** this setting is not recommended when running AMP Optimizer in your backend on every request as execution time can increase by up to 7x.
+- additional dependency (when using amp-script): `npm install terser`
```

Comment:
why are they not included? to keep the bundle size down? May make sense to mention that at some point, somewhere

Reply:
Above I'm saying `*Fast:** this will perform only critical optimizations minimizing bundle install size and execution time. `